client/my-sites/domains/domain-search/index.jsx

```diff
@@ -208,6 +209,10 @@ class DomainSearch extends Component {
 		const { domainRegistrationMaintenanceEndTime } = this.state;
 
 		const hasPlanInCart = hasPlan( cart );
+		// When the sidebar is hidden by launchpad, go back to Launchpad
+		const backButtonUrl = getQueryArgs()?.[ 'is-launchpad-step' ]
+			? 'javascript:history.back()'
```

Comment:
Probably not great to do it this way, just wanted to get something up and working

Reply:
In this case we've landed on the domains page from the Launchpad NUX screen by clicking on customize domain as seen in the screenshots. The idea is to keep them in the Launchpad experience which is why we're hiding the sidebar. It should be easy enough to get the proper URL because this is very fragile 👍